### DISCUSSION TITLE
Website Footer

### DISCUSSION THREAD

**Discussion by Febakke** [digdir/designsystemet#1673] [Patterns for public services]
### Relevant
- https://github.com/digdir/innsiktsbibliotek/issues/32

https://github.com/digdir/designsystemet/issues/1234

**Comment by Camulos** [comment]
This pattern is for the website or page footer, not for the footer of other regions that allow header/footer within.

What information should always be presented in the footer?
Logo, registration number, links to certain types of information?, social media links?
